File: scrape.py
import requests
from bs4 import BeautifulSoup

url = "https://www.lyrics.com/lyric/35065270/Imagine+Dragons/Thunder"
page = requests.get(url)
soup = BeautifulSoup(page.content, 'html.parser')

result = soup.find('pre', id='lyric-body-text').get_text()
print(result)

File: scrape.js
const cheerio = require('cheerio');

const url = 'https://www.lyrics.com/lyric/35065270/Imagine+Dragons/Thunder';

fetch(url)
  .then((response) => response.text())
  .then((html) => {
    const $ = cheerio.load(html);

    const result = $('pre#lyric-body-text').text();
    console.log(result);
  })
  .catch((error) => {
    console.error(`Error fetching the page: ${error}`);
  });
